requirements_dev.txt

```diff
@@ -1,14 +1,17 @@
 pytest>=6.0
+pytest-runner
+pytest-cov
 flake8
+tox
 pytest-flake8
 ipython
 pytest-notebook
 nbsphinx
 nbval>=0.9.6
 nbconvert
-sphinx>=1.7
-bumpversion
-twine
+sphinx>=1.8.5
+bump2version
+Click
```

Comment:
Is `click` not installed by default with `emu`? Until we move to a new packaging system, we probably still need `twine` for package uploads.